oxd-server/src/main/java/org/gluu/oxd/server/RestResource.java

```diff
@@ -44,6 +49,20 @@ public String healthCheck() {
         return "{\"status\":\"running\"}";
     }
 
+    @GET
+    @Path("/get-request-object-jwt/{request_object_id}")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getRequestObject(@PathParam("request_object_id") String id) {
+        return process(CommandType.GET_REQUEST_OBJECT_JWT, convertParamToJsonString(GetRequestObjectJwtParams.class, id), GetRequestObjectJwtParams.class, null, null, httpRequest);
```

Comment:
Not critical but I guess we can remove `convertParamToJsonString` and `GetRequestObjectJwtParams` and introduce `StringParam` which should hold simple string value and should be handled in special way.
```
class StringParam implements IParam {
    String value;   
}
```

`/get-request-object-jwt` -> `/get-request-object`

Reply:
Done